webapp/sources/utils/src/test/scala/com/normation/ZioCommonsTest.scala

```diff
@@ -504,11 +504,13 @@ import _root_.zio.internal._
           val blockingExecutor: UIO[Executor] = Executor.fromExecutionContext(Int.MaxValue)(scala.concurrent.ExecutionContext.global).succeed
         }
       }
-      override val platform: Platform = PlatformLive.fromExecutionContext(scala.concurrent.ExecutionContext.global) //.withTracingConfig(_root_.zio.internal.tracing.TracingConfig.disabled)
-      override val environment = new Clock.Live with Console.Live with ZSystem.Live with Random.Live with Blocking.Live
+      override val Platform: Platform = PlatformLive.fromExecutionContext(scala.concurrent.ExecutionContext.global) //.withTracingConfig(_root_.zio.internal.tracing.TracingConfig.disabled)
+      override val Environment = new Clock.Live with Console.Live with ZSystem.Live with Random.Live with Blocking.Live
+      def platform = Platform
+      def environmet = Environment
```

Comment:
isn't there a typo ?

Reply:
it should not be there at all actually :)